### PR TITLE
Prepare for release v1.1.0

### LINKED ISSUE #51: Update: Change the signing keys
In v1.1.0

Add info to release comment.

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
closing #53 
closing #52

### CODE REVIEW

**Comment by lukasmatusiewicz** [comment]
closing #51

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## 1.1.0 02/2024
+
+### Features
+* Poll in browser
+
+### Fixes
+* Separate the css and js from the view file
+* Fix the sealing violation issue occurred on jetty
+* Compatibility with the IdP 5.0.0 and higher
```
Comment (nilsbehlen) [inline changelog.md:9]: If it is not compatible with IdP 4.x.x anymore, we should state that here just to be clear.

File: privacyIDEA-impl/src/main/java/org/privacyidea/action/PrivacyIDEAAuthenticator.java [hunk 2]
```diff
@@ -67,9 +82,9 @@ else if (piContext.getWebauthnSignResponse() != null && !piContext.getWebauthnSi
         }
         else if (piContext.getMode().equals("otp"))
         {
-            if (request.getParameterValues("pi_otp_input") != null)
+            if (request.getParameterValues("otp") != null)
             {
-                String otp = request.getParameterValues("pi_otp_input")[0];
+                String otp = request.getParameterValues("otp")[0];
```
Comment (nilsbehlen) [inline privacyIDEA-impl/src/main/java/org/privacyidea/action/PrivacyIDEAAuthenticator.java:87]: is it safe to access the first element without checking if the array(?) is empty?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # shibboleth-plugin
 
 ### Min. shibboleth-idp version:
-4.3.x
```
Comment (nilsbehlen) [inline README.md:4]: We should also state here that the main branch is idp 5 compatible. The version for 4 will be somewhere (insert instructions)

File: privacyIDEA-impl/src/main/java/org/privacyidea/action/PrivacyIDEAAuthenticator.java (hunk 2 above)
Comment (lukasmatusiewicz) [reply privacyIDEA-impl/src/main/java/org/privacyidea/action/PrivacyIDEAAuthenticator.java:87]: [18682b4](https://github.com/privacyidea/shibboleth-plugin/pull/50/commits/18682b4391dd678c14981a4be0dcfcf4dbedf34e)

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**